### PR TITLE
feat: ui changes

### PR DESCRIPTION
**Pull request by kasin-it**

### CODE REVIEW

File: starters/shopify-meilisearch/components/footer.tsx [hunk 1]
```diff
@@ -1,154 +1,158 @@
...
-        </footer>
+        </div>
       </div>
-    </div>
+    </footer>
   )
 }
 
-function FacebookIcon(props: React.SVGProps<SVGSVGElement>) {
+function FacebookIcon(props) {
```
Comment (ddaoxuan) [inline starters/shopify-meilisearch/components/footer.tsx:117]: what's the reason to remove types?

File: starters/shopify-meilisearch/views/homepage/categories-section.tsx [hunk 2]
```diff
@@ -1,62 +1,31 @@
...
-import Image from "next/image"
-import Link from "next/link"
-import { getDemoCategories, isDemoMode } from "utils/demo-utils"
+import { CategoryCard } from "components/category-card"
+import { PlatformCollection } from "lib/shopify/types"
+import { cn } from "utils/cn"
 
-export async function CategoriesSection() {
-  const categories = await getCategories()
+export async function CategoriesSection({ categories }: { categories: Pick<PlatformCollection, "handle" | "title" | "id" | "image">[] }) {
```
Comment (ddaoxuan) [inline starters/shopify-meilisearch/views/homepage/categories-section.tsx:5]: This component could just fetch categories on its own

File: starters/shopify-meilisearch/views/product/product-added-alert.tsx [hunk 3]
```diff
@@ -17,43 +17,34 @@ export function ProductAddedAlert({ className }: { className?: string }) {
...
             </div>
           </div>
+          <span className="text-xs">
+            {mapCurrencyToSign(combination!.price!.currencyCode as CurrencyType)}
+            {parseInt(combination!.price!.amount!, 10).toFixed(2)}
+          </span>
         </div>
         <div className="flex items-center justify-between gap-2">
-          <Button onMouseEnter={preloadSheet} onClick={openCart} className="items-center justify-center">
+          <Button onMouseEnter={preloadSheet} onClick={openCart} variant={"outline"} className="bg-white transition-all hover:scale-105">
```
Comment (ddaoxuan) [inline starters/shopify-meilisearch/views/product/product-added-alert.tsx:44]:
```suggestion
          <Button onMouseEnter={preloadSheet} onClick={openCart} variant="outline" className="bg-white transition-all hover:scale-105">
```

File: starters/shopify-meilisearch/views/product/product-added-alert.tsx [hunk 4]
```diff
@@ -17,43 +17,34 @@ export function ProductAddedAlert({ className }: { className?: string }) {
...
             View cart
           </Button>
-          <Button onClick={() => router.push(cart?.checkoutUrl!)} className="items-center justify-center">
+          <Button variant={"default"} onClick={() => router.push(cart?.checkoutUrl!)} className="rounded-md px-10 py-4 transition-all hover:scale-105">
```
Comment (ddaoxuan) [inline starters/shopify-meilisearch/views/product/product-added-alert.tsx:47]:
```suggestion
          <Button variant="default" onClick={() => router.push(cart?.checkoutUrl!)} className="rounded-md px-10 py-4 transition-all hover:scale-105">
```

File: starters/shopify-meilisearch/views/product/review-button.tsx [hunk 5]
```diff
@@ -14,6 +14,8 @@ export const ReviewButton = ({ productId }: ReviewButtonProps) => {
 
   return (
     <Button
+      variant={"outline"}
```
Comment (ddaoxuan) [inline starters/shopify-meilisearch/views/product/review-button.tsx:17]:
```suggestion
      variant="outline"
```

File: starters/shopify-meilisearch/views/homepage/categories-section.tsx (hunk 2 above)
Comment (kasin-it) [reply starters/shopify-meilisearch/views/homepage/categories-section.tsx]: done

File: starters/shopify-meilisearch/components/footer.tsx (hunk 1 above)
Comment (kasin-it) [reply starters/shopify-meilisearch/components/footer.tsx]: I probably formatted something with the chatbot and had to delete it. I fixed it.

**Pull request merged**